jQuery/honeyHeadquarters.js

```diff
@@ -1,4 +1,27 @@
 // $.ajax('https://sheetsu.com/apis/v1.0/64b5c3f8').done(function(data) {
-$.ajax('http://localhost:3000/citizens').done(function(data) {
-  console.log(data);
+$.ajax('http://localhost:3000/citizens').done(function(citizens) {
+  function getUniqueValuesForField(field, citizens) {
+    const values = ['All'];
+    citizens.forEach((c) => {
+      let v = c[field];
+      if (!values.includes(v)) {
```

Comment:
`Array.includes` has no support in [internet explorer](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes)

If you need a browser friendly way to check, you can use `if(values.indexOf(v) > -1)` or `if(!~values.indexOf(v))`, the `~` operator is a bit shifter. makes a -1 a 0, makes any other number a non-zero number, which evaluates to true.